Topics/starting-a-data-project.md

```diff
@@ -0,0 +1,254 @@
+# Starting a Data Project
+
+# What you will learn 
+In this guide, you'll learn to prepare yourself to start the project. 
+
+This will consist of learning how to look for sources that can help you solve the problem, building the project structure, build good documentation, secure your code, our data and more!
+
+In this guide we'll make heavy use of an awesome best practices collection from the [PLOS | Public Library of Science](https://www.plos.org/). 
+
+Do you know the happiness when you find a _giant diamond cluster in Minecraft_? 
+
+If yes, you need to know that you've just found a fat one:
```

Comment:
> you've just found a fat one:
you've hit a jackpot... --- something in those lines

Reply:
Not sure, can you explain?

Reply:
Again I'm suggesting a new and better sentence.

Your version:
> you've just found a fat one:

Not heard of the above one and it might not be fully clear I think.

My suggestion:
> you've hit a jackpot... --- something in those lines